website/src/content/blog/biome-v1-6.mdx

```diff
@@ -0,0 +1,273 @@
+summary: |
+  Partial support for Astro, Svelte and Vue files, both CLI and LSP.
+  Now, the `extends` field is able to pick configuration files that are exported from a dependency.
+  The formatter has new options.
+authors:
+  - ema
+  - team
+pubDate: 2024-03-08
+coverImage:
+  lightSrc: "@/assets/blog/roadmap-2024/banner-light.png"
+  darkSrc: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import { Image } from "astro:assets";
+import astroLint from "../../assets/blog/biome-v1-6/astro-linter.png"
+import vueLint from "../../assets/blog/biome-v1-6/vue-linter.png"
+import svelteLint from "../../assets/blog/biome-v1-6/svelte-linter.png"
+
+Update Biome using the following commands:
+
+```shell
+npm install --save-dev --save-exact @biomejs/biome@latest
+npx @biomejs/biome migrate
+```
+
+## Partial support for Astro, Svelte and Vue files
+
+In this release, we're happy to provide partial support for Astro, Svelte and Vue files. What does **partial** support mean?
+
+While the team is working on a [unified data structure](https://github.com/biomejs/biome/discussions/1726) for HTML-ish languages, we discovered that we could provide Biome functionalities to those files with just a few changes, albeit with some limitations.
+
+This means that Biome is able to analyze the JavaScript/TypeScript portion of said files, and all features are available: formatting, linting and import sorting! Here's an example of what you should expect in terms of developer experience:
+
+<Tabs>
+    <TabItem label="Astro" icon="astro">
+        <div align="center">
+            <Image src={astroLint} alt="Screenshot of Biome linting in action for an Astro file in VSCode" width="500"  />
+        </div>
+    </TabItem>
+    <TabItem label="Vue" icon="seti:vue">
+        <div align="center">
+            <Image src={vueLint} alt="Screenshot of Biome linting in action for an Vue file in VSCode" width="500"  />
+        </div>
+    </TabItem>
+    <TabItem label="Svelte" icon="seti:svelte">
+        <div align="center">
+            <Image src={svelteLint} alt="Screenshot of Biome  linting in action for an Svelte file in VSCode" width="500"  />
+        </div>
+    </TabItem>
+</Tabs>
+
+Make sure to read the [documentation about expectations and limitations](/internals/language-support#html-super-languages-support).
+
+## Configuration, lighter and more powerful
+
+### Support for `biome.jsonc`
+
+Biome now accepts the file `biome.jsonc` as configuration! Now, you can insert all the comments you want in a configuration.
+
+### `extends` resolves dependencies
+
+From this version, Biome can use the `extends` property to *resolve* other configuration files that are inside installed dependencies.
```

Comment:
```suggestion
From this version, Biome can use the `extends` property to _resolve_ other configuration files that are inside installed dependencies.
```

Reply:
I don't understand the suggestion, the emitted markup is the same?

Reply:
Yes. It is just more conventional to use `_it_` for italics.